app/models/lettings_log.rb

```diff
@@ -538,6 +539,7 @@ def reset_derived_questions
       end
     end
   end
+  # show in diff
```

Comment:
the first entry in dependent questions above says that:
if we change "Is this letting a renewal?" to "No" (renewal == 0) then we can assume that it was previously "Yes" (1) and clear the field that would have been derived from that state.
So far, this way works, however if we are ever to make derivations from an answer to a radio question with more than two answers, we will need to know what state we are moving away from, rather than moving to in order to know whether we can clear the dependent fields.
eg if we derive a value from something like relat3 being "C" for child in the future, then in order to clear the derived value, we need to know in future if `relat3_was == "C"` rather than `relat3 == "P"/"X"/etc`